cloud/index.rst

```diff
@@ -23,12 +23,125 @@ The key items you'll care about for each node:
 - Memory - The memory on each node varies based on the size of node you select
 - Cores - The cores on each node varies based on the size of node you select
 
-High Availability
-~~~~~~~~~~~~~~~~~
+Connecting to Citus Cloud
```

Comment:
Should we maybe rename the current security section to "Connection and Security" and move both these sections in there? Any other options?